browser/TemplateProvider.js

```diff
@@ -30,60 +30,44 @@
 
 'use strict';
 
-module.exports = TemplateProvider;
-
-/**
- * Creates new instance of Handlebars template provider.
- * @param {Handlebars} $handlebars Handlebars factory.
- * @constructor
- */
```

Comment:
We need to move this jsdoc above to a constructor and change the parameters description.